src/controllers/engine/controllerengine.cpp

```diff
@@ -527,7 +481,7 @@ void ControllerEngine::errorDialogButton(const QString& key, QMessageBox::Standa
                SLOT(errorDialogButton(QString, QMessageBox::StandardButton)));
 
     if (button == QMessageBox::Retry) {
-        emit(resetController());
+        reloadScripts();
```

Comment:
No slot was connected to this signal. We can reintroduce the signal in the future if other parts of Mixxx need to listen to this.

Reply:
Okay. Would you mind just commenting it out here then so it acts as a placeholder?

Reply:
Sure I can add a comment.

Reply:
Well, actually maybe we better don't. It might be confusing because:
"resetController" is not an 'event' generated by ControllerEngine. It's an action that we could take in response to the `fileChanged` signal, which already exists and is connected to `scriptHasChanged` slot.

So I'd say that the signal is already there, it's `fileChanged` which is already public so anybody can connect to it.

Reply:
That's okay for now. (It's important to note that a file changing is not necessarily the same as calling for a full controller reset, but not a big deal to add such a signal later if we do ever need it.) Thanks for your patience with me. :)